helpers.py

```diff
@@ -34,3 +34,12 @@ def setup_logger():
     console.setFormatter(logging.Formatter(log_format, datefmt=log_datefmt))
     logging.getLogger('').addHandler(console)
     logging.getLogger('urllib3').setLevel(logging.WARNING)
+
+
+def sign_message_with_static_key(message):
+    from eth_keys import keys
```

Comment:
Is it better to import this in method than at the top of this file?

Reply:
done